### ISSUE TITLE
Cloud Hosting Providers

### ISSUE THREAD

**Issue by atmosmaciel** [levi-ufma/levi-api#1]
Providers list Cloud Hosting:

http://digitalocean.com

https://www.linode.com/

https://www.vultr.com/

https://aws.amazon.com/pt/

https://cloud.google.com

**For test i'm suggest:**

https://www.heroku.com/

@jherfson assim que puder faz a análise aí e define qual o melhor. Pode fechar a issue depois disso.

**Comment by jherfson** [comment]
Certo. Vou ver com @RodolphoMouta esses dias. Lembrando que vamos marcar a reunião com o pessoal da TI. Provalvemente será na próxima semana.
